Week1/homework 1/app.js

```diff
@@ -0,0 +1,156 @@
+'use strict';
+//1-  Use the mysql package (https://www.npmjs.com/package/mysql)
+
+//2-  Make a connection to MySQL, using your MySQL root login credentials
+
+const mysql = require('mysql');
+const dataOfCities = require('./city.json');
+const dataOfCountries = require('./country.json');
+
+// create connection 
+const db = mysql.createConnection({
+    host     : 'localhost',
+    user     : 'root',
+    password : 'mosleh1234',
+    database : 'world'
+});
+
+// connect 
+db.connect((err) => {
+    if(err) {
+        throw err;
+    }
+    console.log('Mysql connected');
+});
+//3-  Write JavaScript code that incorporates SQL commands for the following:
+// 3-1 Create database 
+let sql = 'CREATE DATABASE IF NOT EXISTS world';
+
+
+// execute queries 
+const execQuery = function(table){
+    db.query(table, (err, result) => {
+        if (err) throw err;
+        
+});
+}
+
+// 3-2 Create countries table and insert data into it
+async function createCountriesTableAndInsertData(){
+const CREATE_TABLE_COUNTRIES = `CREATE TABLE IF NOT EXISTS countries (
+    ID int not null auto_increment primary key,
+    name varchar(254) not null,
+    continent enum('Asia', 'Africa', 'North America', 'South America', 'Oceania', 'Europe', 'Antarctica'),
+    region varchar(254),
+    surface_area float,
+    indep_year int,
+    population int,
+    life_expectancy float,
+    gnp float,
+    gnp_old float,
+    local_name varchar(254),
+    government_form varchar(254),
+    head_of_state varchar(254),
+    capital int
```

Comment:
I think in this case int is not relevant. It is ok if you add a foreign key to reference cities but you should see this tmw.